src/Bicep.LangServer/Completions/ModuleReferenceCompletionProvider.cs

```diff
@@ -86,7 +86,8 @@ public async Task<IEnumerable<CompletionItem>> GetFilteredCompletions(Uri source
         // I.e. typing with an empty path:  module m1 <CURSOR>
         private IEnumerable<CompletionItem> GetTopLevelCompletions(BicepCompletionContext context, string replacementText, Uri sourceFileUri)
         {
-            if (!context.Kind.HasFlag(BicepCompletionContextKind.ModulePath))
+            if (!context.Kind.HasFlag(BicepCompletionContextKind.ModulePath) &&
+                !context.Kind.HasFlag(BicepCompletionContextKind.UsingFilePath))
```

Comment:
Now that a registry reference can be supplied for `using` statements, do these completion context kinds need to be separate? (This is maybe tech debt for a separate PR)

Reply:
They could be combined, but I also don't have any hard feelings about leaving them separate - this is at least explicit, and also possible we may want the behaviors to diverge in future.